src/DazCli.js

```diff
@@ -11,9 +12,12 @@ class DazCli extends Daz {
     _run(args, input, output, terminal) {
         if (args.length >= 3) {
             const command = args[2]
-    
+
             if (command === '--help') {
-                this.output.write(`${fs.readFileSync(path.join(__dirname, 'help.txt'))}\n`)
+                exec(`sh ${__dirname}/help.sh`,
```

Comment:
USE 'path' here instead, instead, to securely resolve routes (that is: \`sh ${path.join(__dirname, 'help.sh')}\`

Reply:
ohh! I thought that it wouldn't work with `path.join` for this case, but i'm testing and I can see that it works fine hehe 😁